### DISCUSSION TITLE
How to get a development license?

### DISCUSSION THREAD

**Discussion by connessionetech** [ant-media/Ant-Media-Server#5554] [General]
A few months back there was [an announcement on developer license](https://community.antmedia.io/topic/1460-developer-license-those-who-need-a-developer-license-can-contact-me/#comment-5265). I would like to know where does this stand now? How can I request a developer license without limitations to try out the server and development on it?

**Comment by mekya** [comment]
Hi @connessionetech,
Because you're a very valuable community member, we're more than happy to give you a license for your tests/development and experiments. I think my colleagues has already provided it to you. We discuss how we cooperate together more actively if you want to do so.
